plugin.audio.bbcpodcasts/resources/settings.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
That's the old format for settings. See https://kodi.wiki/view/Add-on_settings_conversion.